packages/common/src/ui/components/buttons/FlatButton.tsx

```diff
@@ -10,27 +10,39 @@ interface ButtonProps {
   disabled?: boolean;
 }
 
-const StyledButton = styled(MuiButton)({
+const StyledButton = styled(MuiButton, {
+  shouldForwardProp: prop => prop !== 'isRtl',
+})<{ isRtl: boolean }>(({ isRtl }) => ({
   fontWeight: 700,
   marginLeft: 5,
   marginRight: 5,
   textTransform: 'none' as Property.TextTransform,
-});
+  '& .MuiButton-startIcon': isRtl
```

Comment:
The MuiButton is adding an 'rtl' style but this has the same margins on it as the ltr version 😕 
am overriding manually with the values for left and right reversed